### PR TITLE
iteration_1_sales_analyst

### PR DESCRIPTION
**Pull request by DesTodo**
Please review test set up for initialize on line 6; more will be pushed onto this branch tonight, please don't erase yet. TY :D

### CODE REVIEW

**Comment by emcooper** [comment]
Ok great 👍 , I'm just going to wake up early and look at this in the morning and then get started on my part. Don't work too late!

**Comment by DesTodo** [comment]
I created a few more methods and pushed onto branch AFTER the initial pull request. Seems like these updates were added to the original PR. Please review. Tnx!

File: lib/sales_analyst.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+require'pry'
+
+class SalesAnalyst
+  attr_reader :se
+
+  def initialize(se)
+    @se = se
+  end
+
+  def average_items_per_merchant
```
Comment (DesTodo) [inline lib/sales_analyst.rb:11]: Another way to build def average_items_per_merchant is: 

number_items = se.items.items.count
number_merchants = se.merchants.merchants.count
average = number_items.to_f / number_merchants
return average.round(2)

PRO: Cuts test time in half.
CON: No array with merchant item count for later use.

File: lib/sales_analyst.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    total_merchant_count = se.merchants.merchants.count
+    total_items_count    = items_per_merchant.reduce(:+).to_f
+    average              = total_items_count / total_merchant_count
+    return average.round(2)
```
Comment (DesTodo) [inline lib/sales_analyst.rb:15]: Moving .to_f to total_items_count from average, changed the return value from 2.0 to 2.8778... 

Adding .round(2) rounds to => 2.88 to match the value in the spec.

File: lib/sales_analyst.rb (hunk 1 above)
Comment (emcooper) [reply lib/sales_analyst.rb:11]: Ok, whatever you think is best given the other methods. I'll merge this for now but we can always modify.

**Review by emcooper — APPROVED** [review]
Looks good 👍. Glad you finally got to use reduce!

**Pull request merged**